src/hooks/useSession.tsx

```diff
@@ -12,14 +12,13 @@ const emptyUser: User = {
 
 export type SessionReturn = {
   accessToken: string,
-  user: User;
-};
+} & Auth0ContextInterface<User>;
 
 /**
  * This hook captures Authorization data from Google Oauth2
  */
 export default function useSession(): SessionReturn {
-  const { user, isAuthenticated } = useAuth0();
+  const auth0 = useAuth0();
```

Comment:
small refactor so `useSession` returns everything that `useAuth0` returns instead of having to expand manually